src/App.js

```diff
@@ -10,6 +10,8 @@ import { search } from "fast-fuzzy";
 import Details from "./pages/Details";
 import { useNavigate } from "react-router-dom";
 import useLocalStorage from "./components/hooks/useLocalStorage";
+import PersonalData from "./pages/PersonalData";
+import Bestellung from "./pages/Bestellung";
```

Comment:
Please use English for coding and naming variable and components!!

Reply:
already renamed Bestellung, thanks!